image/nginx/nginx.conf

```diff
@@ -0,0 +1,100 @@
+user www-data;
```

Comment:
This would pertain to situations where you're running with a VM (OSX, Windows) and using the vboxsf filesystem driver to point back to your filesystem, live.

I think we spoke earlier about how to bind-mount directories for development mode.  Can you verify or just mention here whether this line is okay when you're bind-mounted (not just running off of files that were added to the container)?